changelog/fix_parentheses_in_forwarded_args_for_style_method_call_with_args_parentheses.md

```diff
@@ -0,0 +1 @@
+* [#9627](https://github.com/rubocop/rubocop/issues/9627): Allow parentheses for forwarded args in `Style/MethodCallWithArgsParentheses`'s `omit_parentheses` style to avoid endless range ambiguity. ([@gsamokovarov][])
```

Comment:
This looks like a typo :-)

```suggestion
* [#9637](https://github.com/rubocop/rubocop/pull/9637): Allow parentheses for forwarded args in `Style/MethodCallWithArgsParentheses`'s `omit_parentheses` style to avoid endless range ambiguity. ([@gsamokovarov][])
```

Reply:
So I didn't predict it after all 😅. Thanks for noticing!